include/zephyr/sys/mem_manage.h

```diff
@@ -233,10 +233,10 @@ extern "C" {
  * This API is part of infrastructure still under development and may
  * change.
  *
- * @param virt [out] Output virtual address storage location
- * @param phys Physical address base of the memory region
- * @param size Size of the memory region
- * @param flags Caching mode and access flags, see K_MAP_* macros
+ * @param[out] virt Output virtual address storage location
```

Comment:
doxygen doesn't "build" the doc for "z_" symbols but you may still want to correct:

```suggestion
 * @param[out] virt_ptr Output virtual address storage location
```

Reply:
Fixed.